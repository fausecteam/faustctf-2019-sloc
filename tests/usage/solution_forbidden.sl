DECLARE $fd
DECLARE $str
CALL FOPEN "db.sqlite" "r" -> $fd
LABEL reading
	CALL READCHAR $fd -> $str
	IF $str E -1 GOTO end
	CALL WRITECHAR $str
	GOTO reading
LABEL end
