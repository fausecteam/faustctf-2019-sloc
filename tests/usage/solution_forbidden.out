just_a_dummy
